### PR TITLE
Use SimpleFormatter.formatMessage to replace {0} with actual parameters

### LINKED ISSUE #5801: Parameter formatting is broken with TruffleLoggers
Logs coming out of TruffleLogger don't use the provided arguments leading to incomplete messages:
```
[debug] [...] [epb.org.enso.interpreter.epb.EpbContext] Initializing languages {0}
[trace] [...] [epb.org.enso.interpreter.epb.EpbContext] Starting initialization thread
[trace] [...] [epb.org.enso.interpreter.epb.EpbContext] Entering initialization thread
[trace] [...] [epb.org.enso.interpreter.epb.EpbContext] Initializing language {0}
[trace] [...] [epb.org.enso.interpreter.epb.EpbContext] Initializing on background
[debug] [...] [enso] Executing commands in a separate command pool
[debug] [...] [epb.org.enso.interpreter.epb.EpbContext] Done initializing language {0} with {1} in {2} ms
```
Notice the `{0}` for parameter position which should have been replaced with the correct argument.

As per logs in https://github.com/enso-org/enso/blob/develop/engine/runtime-language-epb/src/main/java/org/enso/interpreter/epb/EpbContext.java#L66 or https://github.com/enso-org/enso/blob/develop/engine/runtime-language-epb/src/main/java/org/enso/interpreter/epb/EpbContext.java#L80.

Those logs were introduced in https://github.com/enso-org/enso/pull/5680 but I can see plenty of places where we could use parameter formatting except we can't so logs are full of String concatenation (also affecting performance).

### PR DESCRIPTION
**Pull request by JaroslavTulach**
### Pull Request Description

Fixes #5801 to properly format Truffle log records before sending them for further processing.

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [x] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- All code has been tested:
  - [x] Unit tests have been written where possible.

### CODE REVIEW

File: lib/scala/logging-service/src/main/scala/org/enso/loggingservice/JavaLoggingLogHandler.scala [hunk 1]
```diff
@@ -11,6 +11,7 @@ class JavaLoggingLogHandler(
   levelMapping: Level => LogLevel,
   connection: LoggerConnection
 ) extends Handler {
+  private val formatter = new SimpleFormatter()
```
Comment (hubertp) [inline lib/scala/logging-service/src/main/scala/org/enso/loggingservice/JavaLoggingLogHandler.scala]: This should likely be in the companion object of `JavaLoggingLogHandler` class

**Review by radeusgd — APPROVED** [review]

File: lib/scala/logging-service/src/main/scala/org/enso/loggingservice/JavaLoggingLogHandler.scala (hunk 1 above)
Comment (JaroslavTulach) [reply lib/scala/logging-service/src/main/scala/org/enso/loggingservice/JavaLoggingLogHandler.scala]: Companion, I see: d79e1476b4

File: build.sbt [hunk 2]
```diff
@@ -873,6 +873,8 @@ lazy val `logging-service` = project
       akkaHttp,
       "io.circe"              %%% "circe-core"   % circeVersion,
       "io.circe"              %%% "circe-parser" % circeVersion,
+      "junit"               % "junit"                 % junitVersion      % Test,
```
Comment (hubertp) [inline build.sbt]: something looks odd with formatting here

**Review by hubertp — APPROVED** [review]

File: build.sbt (hunk 2 above)
Comment (JaroslavTulach) [reply build.sbt]: I've executed:
```
sbt> scalafmtAll
```
I don't know how to format Scala in another way.

Comment (4e6) [reply build.sbt]: there is `scalafmtSbt` command for that

Comment (4e6) [reply build.sbt]: Turned out the SBT sources were never formatted. I created a separate PR for that #5880

Comment (JaroslavTulach) [reply build.sbt]: Reformatted in 0955b86313 - I hope there shall be no clashes with #5880 as the `build.sbt` was properly formatted before this PR.

**Pull request merged**